virtcontainers/qemu_ppc64le.go

```diff
@@ -83,6 +83,9 @@ func newQemuArch(config HypervisorConfig) qemuArch {
 	}
 
 	q.handleImagePath(config)
+
+	q.memoryOffset = config.MemOffset
```

Comment:
Question about the offset here, would this not be different for every device? I dont see where you deduce this based on the size of the block device?

Reply:
This size is set in the config file.
I add it in the new version.